pkg/acp/envoy.go

```diff
@@ -144,3 +151,19 @@ func (w *EnvoyWatcher) IsAlive() bool {
 func (w *EnvoyWatcher) IsReady() bool {
 	return w.IsAlive()
 }
+
+func getDefaultReadyUrl() string {
+	var readyPort int
+	var err error
+	strReadyPort := os.Getenv("AMBASSADOR_READY_PORT")
+	if strReadyPort != "" {
+		readyPort, err = strconv.Atoi(strReadyPort)
+		if err != nil {
+			dlog.Infof(context.Background(), "Unable to parse AMBASSADOR_READY_PORT: %s", err)
+		}
+	}
+	if readyPort < 1 || readyPort > 32767 {
+		readyPort = 8002
+	}
```

Comment:
Suggestions `strconv.ParseUint(strReadyPort, 10, 15)` would do this bounds checking for you.

Reply:
Yeah I would prefer to have an explicit bounds check. @LukeShu should I resolved this comment?